scripts/lib/CIME/XML/env_mach_specific.py

```diff
@@ -85,6 +85,24 @@ def load_env(self, compiler, debug, mpilib):
         if (envs_to_set is not None):
             self.load_envs(envs_to_set)
 
+
+        self._get_resources_for_case(compiler, debug, mpilib)
+
+    def _get_resources_for_case(self, compiler, debug, mpilib):
+        resource_nodes = self.get_nodes("resource_limits")
+        if resource_nodes is not None:
+            nodes = self._compute_resource_actions(resource_nodes, compiler, debug, mpilib)
+            for name, val in nodes:
+                try:
```

Comment:
What's the point of this try: if you're just going to re-raise the exception?

Reply:
It's a place holder for figuring out something better - I'll remove it.